tasks/main.yml

```diff
@@ -98,6 +98,7 @@
     owner: root
     group: root
     mode: 0640
+    validate: "{{ keepalived_install_prefix }}/keepalived --use-file=%s --check --all-config --config-test"
```

Comment:
Do you have an idea if the work in "all" versions? Or is this recently added to `keepalived`.

Reply:
I modified the validate command after testing with molecule and tested following versions:

**From Repo:**
- Debian 10: `Keepalived v2.0.10 (11/12,2018)`
- Debian 11: `Keepalived v2.1.5 (07/13,2020)`
- Ubuntu 18.04: `Keepalived v1.3.9 (10/21,2017)`
- Ubuntu 20.04: `Keepalived v2.0.19 (10/19,2019)`

**Git:**
- Debian 11: `Keepalived v2.2.4 (08/21,2021)`